bindings/protocols/coap/context.jsonld

```diff
@@ -0,0 +1,43 @@
+{
+    "@context": {
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "cov": "https://www.w3.org/2019/wot/coap#",
+        "method": {
+            "@id": "cov:hasMethod",
```

Comment:
To be honest, I mostly tried to replicate what was done for Modbus and MQTT in this context file – I guess for the future, the process of creating the JSON-LD @context file for a protocol binding needs to be better documented and/or automated, especially for making it easier for third parties to create a WoT binding.

From my understanding, though, the @context simply links the vocabulary terms (e.g., `cov:method`) to ontology terms (in this case `https://www.w3.org/2019/wot/coap#hasMethod`). So I think this @context does not imply that a form itself is having a method or that it becomes a Request instance (otherwise something similar would also be true for MQTT and Modbus), but I might as well be wrong here. I will take a deeper dive into JSON-LD, but also RDF and SHACL to be able to address your concerns, since some points are still a bit unclear to me as well.